libs/payment/api/src/lib/payment-provider/payrexx-payment-provider.ts

```diff
@@ -183,7 +229,8 @@ export class PayrexxPaymentProvider extends BasePaymentProvider {
     return {
       state,
       paymentID: gateway.referenceId,
-      paymentData: JSON.stringify(gateway)
+      paymentData: JSON.stringify(gateway),
+      customerID: transaction.preAuthorizationId?.toString()
```

Comment:
so the idea is still to return the preAuthorizationId as customerID? I guess we would need to rewrite whole payment provider to change the naming convention etc for the tokenization?

Reply:
Yes, that would be good to do soon. There are so many entry points and abstraction is really poor.
Let's talk it over soon.